Sources/Application/main.swift

```diff
@@ -1 +1,12 @@
-print("Hello")
\ No newline at end of file
+import SnowManLogic
+import Dictionary
+
+let dictionaryManager = DictionaryManager(words: words) 
```

Comment:
You should qualify the `words` with `Dictionary.words` otherwise it's not clear where they come from